reference/ps-modules/PSScriptAnalyzer/PSScriptAnalyzer.md

```diff
@@ -1,10 +1,10 @@
 ---
 Download Help Link: https://aka.ms/ps-modules-help
-Help Version: 1.21.0
+Help Version: 1.22.0
 Locale: en-US
 Module Guid: d6245802-193d-4068-a631-8863a4342a18
 Module Name: PSScriptAnalyzer
-ms.custom: PSSA v1.21.0
+ms.custom: PSSA v1.22.0
```

Comment:
I noticed PSSA is the only module that has the version in the name, should we remove that for consistency?  https://learn.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/overview?view=ps-modules
![image](https://github.com/MicrosoftDocs/PowerShell-Docs-Modules/assets/9250262/b4631ca3-b8b9-4df8-b325-848530dae187)

Reply:
This is intentional. There were discussions about being clear about which version is being documented since we are not maintaining previous versions.

Reply:
Please update the version number in reference/docs-conceptual/toc.yml